packages/portal/docs/style/BootstrapVue/Modal.md

```diff
@@ -0,0 +1,50 @@
+Customised BootstrapVue modal component.
+
+Docs including more features and variants: https://bootstrap-vue.org/docs/components/modal
+
+Used by:
+- ContentWarningModal
+- DebugApiRequests
+- [DownloadFailedModal](/#/Components/Search?id=downloadfailedmodal)
+- DownloadSuccessModal
+- [EntityInformationModal](/#/Components/Entity?id=entityheader)
+- [EntityUpdateModal](/#/Components/Entity?id=entityheader)
+- ErrorModal
+- ItemLikeButton (limit reached)
+- ItemPinButton (limit reached)
+- ItemPinModal
+- [ItemTranscribeButton](/#/Components/Item?id=itemtranscribebutton)
+- SetAddItemModal
+- SetDeleteModal
+- SetFormModal
+- [SetPublicationRequestWidget](/#/Components/Set?id=setpublishbutton)
+- ShareSocialModal ()
+
+default modal
+```jsx
+<div>
+  <b-button v-b-modal.modal-1 class="mb-3 mr-3">Show default modal</b-button>
```

Comment:
The buttons in this modal look misplaced. Is this expected?

Reply:
Yes. We always use a custom modal footer. I propose to see if we can refactor it so the default footer becomes readily usable. There is a subtask for this and other improvements to some of the base components: https://europeana.atlassian.net/browse/EC-6598